shared/src/domain-logic/nexusLocationInNewJersey.ts

```diff
@@ -0,0 +1,13 @@
+import { ProfileData } from "../profileData";
+import { determineForeignBusinessType } from "./businessPersonaHelpers";
```

Comment:
[pebble] Can we change out the relative imports?

Reply:
we don't have relative imports set up for the shared directory